CONTRIBUTING

```diff
@@ -0,0 +1,89 @@
+# Contributing to Dipy
+
+Dipy is an open-source software project, and we have an open development
+process. This means that we welcome contributions from anyone. We do ask that
+you first read this document and follow the guidelines we have outlined here and
+that you follow the [NIPY community code of conduct](http://nipy.org/conduct.html).
```

Comment:
As it says, inspired by others and adapted for our needs. See https://github.com/nipy/nipy.github.com/pull/1